public/views/error-template.html

```diff
@@ -12,7 +12,7 @@
 
     <link rel="stylesheet" href="public/build/grafana.[[ .Theme ]].<%= webpack.hash %>.css">
 
-    <link rel="icon" type="image/png" href="public/img/fav32.png">
+    <link rel="icon" type="image/x-icon" href="public/img/percona-favicon.ico?cb=1584890955588">
```

Comment:
Is there a reason to keep the cache bump tag for the icon?

Reply:
Yes, the old builds contained a pixelated icon with the same name, so this was to make the browser load a new icon versus the cached one.